InvenTree/stock/templates/stock/item_base.html

```diff
@@ -155,18 +155,24 @@ <h4>
         <div class='btn-group'>
             <button id='stock-actions' title='{% trans "Stock adjustment actions" %}' class='btn btn-default dropdown-toggle' type='button' data-toggle='dropdown'><span class='fas fa-boxes'></span> <span class='caret'></span></button>
             <ul class='dropdown-menu' role='menu'>
-                {% if item.can_adjust_location %}
                 {% if not item.serialized %}
+                {% if item.in_stock %}
                 <li><a href='#' id='stock-count' title='{% trans "Count stock" %}'><span class='fas fa-clipboard-list'></span> {% trans "Count stock" %}</a></li>
+                {% endif %}
+                {% if not item.customer %}
                 <li><a href='#' id='stock-add' title='{% trans "Add stock" %}'><span class='fas fa-plus-circle icon-green'></span> {% trans "Add stock" %}</a></li>
+                {% endif %}
+                {% if item.in_stock %}
                 <li><a href='#' id='stock-remove' title='{% trans "Remove stock" %}'><span class='fas fa-minus-circle icon-red'></span> {% trans "Remove stock" %}</a></li>
                 {% endif %}
+                {% if item.in_stock and item.can_adjust_location %}
                 <li><a href='#' id='stock-move' title='{% trans "Transfer stock" %}'><span class='fas fa-exchange-alt icon-blue'></span> {% trans "Transfer stock" %}</a></li>
```

Comment:
The transfer stock button is (incorrectly) part of the `{% if not item.serialized %}` block.

#1448